src/Hook/ActionShopAccountLinkAfter.php

```diff
@@ -33,8 +33,10 @@ class ActionShopAccountLinkAfter extends Hook
      */
     public function execute(array $params = [])
     {
-//        /** @var PsAccountsService $service */
-//        $service = $this->module->getService(PsAccountsService::class);
-//        $service->enableLogin(true);
+//        if ($this->module->isShopEdition()) {
```

Comment:
Can we remove the "isShopEdition()" method? I would prefer an implementation focusing on our business rather than other teams' products ;)